digitalocean/datasource_digitalocean_kubernetes_cluster.go

```diff
@@ -32,6 +32,7 @@ func dataSourceDigitalOceanKubernetesCluster() *schema.Resource {
 			"surge_upgrade": {
 				Type:     schema.TypeBool,
 				Computed: true,
+				Default: true,
```

Comment:
Data sources just report the state returned by the API for existing resources. You probably want to change the resource:

https://github.com/digitalocean/terraform-provider-digitalocean/blob/c1fb04686f1ffd2fc6f49a8de3102f429335dd93/digitalocean/resource_digitalocean_kubernetes_cluster.go#L49

Reply:
@andrewsomething  - I'm not sure if I should change the schema version if I just provide a default value. I have done so. But, let me know if that's necessary.

Reply:
Generally it's not needed unless you are doing a state migration, but there is no problem with bumping it.

https://www.terraform.io/docs/extend/resources/state-migration.html